src/reggae/reggae.d

```diff
@@ -152,7 +152,8 @@ private string[] getJsonOutputArgs(in Options options) @safe {
             return jsonVal.toString;
         }();
 
-        return ["/usr/bin/env", "python", "-B", "-m", "reggae.reggae_json_build",
```

Comment:
Why remove `/usr/bin/env`?

Reply:
I kept it initially, but couldn't get rid of an if statement (choosing either `REGGAE_PATH` or `['/usr/bin/env','python`]`).

`env` shouldn't be necessary; its documentation says that `env` returns the first executable in the path, which is what will happen anyway.

Reply:
My reply is incorrect; `env` returns the first one in `$PATH`. If someone were to have a python executable in the current working directory, this code change would now use it, while previously it should have been ignored.